skimage/filters/thresholding.py

```diff
@@ -1104,16 +1105,24 @@ def threshold_multiotsu(image, classes=3, nbins=256):
 
     Returns
     -------
-    idx_thresh : array
+    thresh_vals : array
```

Comment:
Please replace it with `thresh` only, if we're changing this :) Is this a deprecation @jni?